### PR TITLE
update paragraph and dropdown placeholder

### LINKED ISSUE #173: Change dropdown placeholder wording: "Enter (field) or choose from the dropdown"

### PR DESCRIPTION
**Pull request by mingyokim**
:tickets: **Ticket(s)**: Closes #173 and #168 

---

## :construction_worker: Changes

Just updating the paragraph on the first page and place holder text for `Select` components that are searchable and allow new options

## :thought_balloon: Notes

Any additional things to take into consideration.

## :flashlight: Testing Instructions

Just check for typos i guess

### CODE REVIEW

File: web/components/input/select/index.sass [hunk 1]
```diff
@@ -51,4 +51,5 @@
       img
         width: 12.6px
         height: 10px
-
+.react-select__placeholder
+  white-space: nowrap
```
Comment (chamkank) [inline web/components/input/select/index.sass]: If you do this the hover color gets cut off and the input covers the whole screen width
image

Comment (mingyokim) [reply web/components/input/select/index.sass]: hm doesn't happen to me.. does that happen on master for u too?

image

Comment (chamkank) [reply web/components/input/select/index.sass]: Oh now that you mention it, it also happens on master. You can see if you scroll to the right.

Comment (mingyokim) [reply web/components/input/select/index.sass]: Do you mind if we make a separate issue for it then? Not sure how high priority it is though

Comment (chamkank) [reply web/components/input/select/index.sass]: Yeah no problem, was this code supposed to fix it or does it do something else?

Comment (mingyokim) [reply web/components/input/select/index.sass]: No it just prevents the placeholder text from wrapping on mobile view

Comment (chamkank) [reply web/components/input/select/index.sass]: I guess I'm missing something cause I don't see the placeholder wrapping on either master or this branch. :joy: A screenshot of placeholder wrapping happening on master would be helpful, in the meantime I'm gonna approve this for the placeholder fixes

**Pull request merged**